.gitignore

```diff
@@ -0,0 +1 @@
+x.wasm
```

Comment:
Added to the git ignore for the reasons mentioned in the PR summary.  Please let me know if there is a reason that I am forgetting in favor of checking in the x.wasm file and I can revert this change.